source/tyr/tests/integration/v1_test.py

```diff
@@ -99,7 +99,7 @@ def test_api():
 
 def test_users(create_5_users):
     check_v1_response('users')
-    check_v1_response('users', 'users/{}'.format(create_5_users))
+    check_v1_response('user', 'users/{}'.format(create_5_users))
```

Comment:
So there was an unspotted "bug" before?

Reply:
bug or upgrade, which school is it? We agreed with @kinnou02 that the POST request which returns only 1 user should have a key "user" singular. The request GET /users/user_id also returns only 1 user. So... I changed it to "user" singular. Right or wrong? TBD

Reply:
What we discussed about the singular is for POST and PUT requests. 
For GET, you're right, it's better to leave the key in the plural for homogeneity, even if there's only one result.